veps/vep-6.md

```diff
@@ -0,0 +1,37 @@
+# # VEP-6 Vitess Release cadence
+
+```
+VEP: 6
+Title: Vitess Release Cadence
+Author: Harshit Gangal
+Reviewers: 
+Status: 
+Created: 2024-11-02
+```
+
+## Abstract
+
+[VEP-1](https://github.com/vitessio/enhancements/blob/main/veps/vep-1.md) established a support lifecycle for Vitess. [VEP-2](https://github.com/vitessio/enhancements/blob/main/veps/vep-2.md) later amended parts of it.
+[VEP-5](https://github.com/vitessio/enhancements/blob/main/veps/vep-5.md) modified the yearly major release to 3 and 1 year support for each major release, thus replaces the Support lifecycle section in VEP-1, and all of VEP-2.
+This VEP suggests a change to how often releases are made and its support lifecycle thus replacing all of VEP-5.
+
+All other sections of VEP-1 will continue to be in force without any change.
+
+## Versioning and Release Cadence
+
+The proposed release cycle is best described as _Monotonic Versioning_ with a release cadence similar to Kubernetes.
```

Comment:
similar to Kubernetes is no longer correct. They are still doing 3 a year. It may be ok to delete this entire line. Or just keep the first part of it.

Reply:
done